### PR TITLE
[Refactor] Make changes to some pins

### PR DESCRIPTION
**Pull request by ctt-t**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [x] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/citrus/erdnuss65/info.json [hunk 1]
```diff
@@ -25,81 +30,79 @@
         "rgblight": true,
         "audio": false
     },
-    "ws2812": {
-        "pin": "A15"
-    },
     "rgblight": {
+        "pin": "A15",
```
Comment (drashna) [inline keyboards/citrus/erdnuss65/info.json:34]: This needs to be reverted. rgblight.pin is no longer valid.

File: keyboards/citrus/erdnuss65/keymaps/default/keymap.c [hunk 2]
```diff
@@ -5,19 +5,18 @@
...
 
     [1] = LAYOUT(
-        KC_GRV,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  _______, KC_MPRV,
-        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, KC_PSCR, _______, _______, RESET,   KC_MNXT,
+        KC_TILD,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  _______, KC_MPRV,
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, KC_PSCR, _______, _______, QK_BOOT,   KC_MNXT,
         _______, _______, _______, _______, _______, _______, _______, KC_CALC, _______, _______, _______, _______, _______, KC_HOME,
         _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, KC_MYCM, KC_VOLU, KC_END,
-        _______, _______, _______, _______, _______, KC_MUTE, KC_VOLU, KC_MPLY
+        _______, _______, _______, _______, _______, KC_MUTE, KC_VODN, KC_MPLY
```
Comment (drashna) [inline keyboards/citrus/erdnuss65/keymaps/default/keymap.c:20]: Not a valid keycode?

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/citrus/erdnuss65/info.json (hunk 1 above)
Comment (ctt-t) [reply keyboards/citrus/erdnuss65/info.json:34]: > This needs to be reverted. rgblight.pin is no longer valid.

What does that mean? I'm not sure what to do

Comment (keyboard-magpie) [reply keyboards/citrus/erdnuss65/info.json:34]: https://docs.qmk.fm/#/ChangeLog/20230528 read the top section here

**Comment by keyboard-magpie** [comment]
Per the [PR checklist](https://docs.qmk.fm/#/pr_checklist?id=keyboard-prs):
`keyboard updates and refactors (eg. to data driven) must go through develop to reduce master -> develop merge conflicts` this keyboard refactor/update should be targeting the `develop` branch.

File: keyboards/citrus/erdnuss65/info.json [hunk 3]
```diff
@@ -25,81 +30,81 @@
...
     },
-    "ws2812": {
-        "pin": "A15"
-    },
     "rgblight": {
         "led_count": 1
     },
+    "ws2812": {
+        "pin": "xxx" 
+    },
```
Comment (zvecr) [inline keyboards/citrus/erdnuss65/info.json:38]:
```suggestion
```

File: keyboards/citrus/erdnuss65/info.json [hunk 4]
```diff
@@ -25,81 +30,81 @@
         "rgblight": true,
         "audio": false
     },
```
Comment (zvecr) [inline keyboards/citrus/erdnuss65/info.json:32]:
```suggestion
    },
    "ws2812": {
        "pin": "A15"
    },
```

File: keyboards/citrus/erdnuss65/info.json [hunk 5]
```diff
@@ -7,13 +7,18 @@
...
+        "device_version": "1.0.0"
     },
+
     "matrix_pins": {
-        "rows": ["B10", "B1", "B0", "A7", "A6"],
-        "cols": ["B12", "B14", "B15", "A8", "B13", "B3", "B4", "B5", "A1", "A2", "A0", "A3", "A4", "A5", "B11"]
+        "rows": 
+        ["B10","B1", "B0","A7","A6"],  
+        "cols": 
+        ["B12","B14","B15","B5","B13","B3","B4","B6","A0","A1","A2","A3","A4","A5","B11"]
```
Comment (zvecr) [inline keyboards/citrus/erdnuss65/info.json:17]:
```suggestion
        "rows": ["B10","B1", "B0","A7","A6"],  
        "cols": ["B12","B14","B15","B5","B13","B3","B4","B6","A0","A1","A2","A3","A4","A5","B11"]
```

File: keyboards/citrus/erdnuss65/info.json [hunk 6]
```diff
@@ -25,81 +30,81 @@
...
     "rgblight": {
         "led_count": 1
     },
+    "ws2812": {
+        "pin": "xxx" 
+    },
     "layouts": {
         "LAYOUT": {
             "layout": [
-                {"label": "ESC K00 (B10,B12)", "matrix": [0, 0], "x": 0, "y": 0},
```
Comment (zvecr) [inline keyboards/citrus/erdnuss65/info.json:37]: Ideally the whitespace changes to this section should be reverted.

**Pull request closed without merging**